src/testhost.x86/testhost.x86.csproj

```diff
@@ -14,6 +14,7 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(TargetFramework)' != 'netcoreapp1.0'">
     <RuntimeIdentifier>win7-x64</RuntimeIdentifier>
+    <AutoGenerateBindingRedirects>false</AutoGenerateBindingRedirects>
```

Comment:
Please add a comment why is this required. CrossPlatEngine no longer has a dependency on JSON.net.